src/tasks/publish/index.ts

```diff
@@ -168,17 +178,36 @@ async function publishTask(
     contentUri: apm.toContentUri('ipfs', contentHash)
   }
 
+  const rootAccount = await getRootAccount(bre)
+  const network = await provider.getNetwork()
+  const isTestNetwork = network.chainId > 100
+  if (!managerAddress && isTestNetwork) managerAddress = rootAccount
```

Comment:
I think we should default to this behavior on every network actually.

Reply:
Fixed with https://github.com/aragon/buidler-aragon/pull/90/commits/6ead048c72670b9891d91957932bdccafb437215